pkgs/development/python-modules/django-huey-monitor/default.nix

```diff
@@ -0,0 +1,79 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools-scm,
+  django,
+  huey,
+  bx-django-utils,
+  bx-py-utils,
+  pytestCheckHook,
+  pytest-django,
+  model-bakery,
+  packaging,
+  typeguard,
+  colorlog,
+  manage-django-project,
+  pkgs,
```

Comment:
```suggestion
  bash,
```

Reply:
Would be shadowed, if someone contributed https://pypi.org/project/bash/.